<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QMainWindow" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bernard MT Condensed</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Estructuras de Datos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bernard MT Condensed</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Algoritmos de Ordenamiento</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="quizAlgoritmos">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bernard MT Condensed</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quiz sobre Estructuras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quizOrdenamieto">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>340</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bernard MT Condensed</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quiz sobre Algoritmos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClaseAlgoritmos">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bernard MT Condensed</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clase Educativa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClaseOrdenamieto">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bernard MT Condensed</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clase Educativa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Bienvenido:</string>
    </property>
   </widget>
   <widget class="QLabel" name="iD">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bernard MT Condensed</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Â¿Sobre que quieres aprender hoy?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/images/image002.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/images/orden.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu_Inicial">
    <property name="title">
     <string>Menu Inicial</string>
    </property>
   </widget>
   <addaction name="menuMenu_Inicial"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
